src/main/java/org/generationcp/middleware/dao/LocationDAO.java

```diff
@@ -705,7 +733,7 @@ public Location saveOrUpdate(final Location location) {
 			locations = criteria.list();
 		} catch (final HibernateException e) {
 			throw new MiddlewareQueryException(
-					this.getLogExceptionMessage("getByIds", "", null, e.getMessage(), LocationDAO.CLASS_NAME_LOCATION), e);
+					this.getLogExceptionMessage("getByUniqueID", "programUUID", null, e.getMessage(), LocationDAO.CLASS_NAME_LOCATION), e);
```

Comment:
@cuenyad I guess the function was not renamed, so it should still be `getByIds'

Reply:
@clarysabel the name of the function is getByUniqueID not getByIds. The commit was to fixed that.